OrcScala/src/orc/error/OrcException.java

```diff
@@ -24,7 +24,7 @@
  *
  * @author dkitchin
  */
-public abstract class OrcException extends Exception {
+public abstract class OrcException extends RuntimeException {
```

Comment:
Why? We are Scala in 99% of the time so they are unchecked anyway.

The reason I changed this is that if they are checked I have to wrap them in the truffle interpreter. I could work around this, but I didn't know of any reason to have them checked.